compiler-core/src/type_/error.rs

```diff
@@ -119,7 +119,7 @@ pub enum ModuleSuggestion {
 }
 
 impl ModuleSuggestion {
-    pub fn suggestion(&self, module: &str) -> String {
+    pub fn qualified_suggestion(&self, module: &str) -> String {
```

Comment:
I think `suggest_import` may be more clear. It's not a suggestion that is qualified.